src/quicr_client_raw_session.cpp

```diff
@@ -539,36 +578,31 @@ bool
 QuicRClientRawSession::notify_pub_fragment(
   const messages::PublishDatagram& datagram,
   const std::weak_ptr<SubscriberDelegate>& delegate,
-  const std::map<int, bytes>& frag_map)
+  const std::map<int, bytes>& buffer)
 {
-  if ((frag_map.rbegin()->first & 0x1) != 0x1) {
+  if ((buffer.rbegin()->first & 0x1) != 0x1) {
     return false; // Not complete, return false that this can NOT be deleted
   }
 
   bytes reassembled;
-
   int seq_bytes = 0;
-  for (const auto& item : frag_map) {
-    if ((item.first >> 1) - seq_bytes != 0) {
+  for (const auto& [sequence_num, data] : buffer) {
+    if ((sequence_num >> 1) - seq_bytes != 0) {
       // Gap in offsets, missing data, return false that this can NOT be deleted
       return false;
     }
 
-    reassembled.insert(
-      reassembled.end(), item.second.begin(), item.second.end());
-    seq_bytes += item.second.size();
-  }
+    reassembled.insert(reassembled.end(),
+                       std::make_move_iterator(data.begin()),
+                       std::make_move_iterator(data.end()));
 
-  for (const auto& entry : sub_delegates) {
```

Comment:
Good to remove this as it was old code for the idea that we would have multiple matches.